.github/workflows/plugin-cd.yml

```diff
@@ -94,14 +94,38 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           fetch-depth: "0"
-      - name: ci/download-artifact
+      - name: cd/download-artifact
         uses: actions/download-artifact@65a9edc5881444af0b9093a5e628f2fe47ea3b2e # v4.1.7
         with:
           name: dist
           path: dist
-      - name: ci/publish-release
+      - name: cd/import-gpg-key
+        id: import-gpg-key
+        env:
+          GPG_PRIVATE_KEY_BASE64: ${{ secrets.PLUGIN_GPG_PRIVATE_KEY }}
+        run: |
+          umask 077
+          echo $GPG_PRIVATE_KEY_BASE64 | base64 --decode > private_key.asc
+          gpg --import private_key.asc
+          export KEY_ID=$(gpg --list-secret-key --with-colons | grep '^sec' | cut -d: -f5)
+          echo "::set-output name=key_id::${KEY_ID}"
+      - name: cd/sign-artifacts
+        run: |
+          for artifact in *.tar.gz; do
+            echo -n "Signing $artifact "
+            gpg -u ${{ steps.import-gpg-key.outputs.key_id }} --yes --armor --personal-digest-preferences SHA256 --detach-sign "$artifact"
+            gpg -u ${{ steps.import-gpg-key.outputs.key_id }} --yes --personal-digest-preferences SHA256 --detach-sign "$artifact"
+            echo "OK!"
+          done
+        working-directory: dist
+      - name: cd/publish-release
         run: |
-          gh release create "$GITHUB_REF_NAME" --title "$GITHUB_REF_NAME" --notes-file release-notes.md  *.tar.gz
+          gh release create "$GITHUB_REF_NAME" --title "$GITHUB_REF_NAME" --notes-file release-notes.md *.tar.gz *.tar.gz.sig *.tar.gz.asc
         working-directory: dist
         env:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+      - name: cd/cleanup
+        run: |
+          gpg --delete-secret-key ${{ steps.import-gpg-key.outputs.key_id }}
+          rm -f private_key.asc
+        working-directory: dist
```

Comment:
`private_key.asc` file is not under dist/ directory . This will probably fail . 
Need to remove the `working-directory` directive

Reply:
fixed 👍